kafka_service/kafka/kafka_consumer.go

```diff
@@ -0,0 +1,66 @@
+package kafka
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/smtp"
+	"os"
+	"strings"
+
+	"github.com/gokhankocer/TODO-API/entities"
+	"github.com/segmentio/kafka-go"
+)
+
+func SendEmail(message string, toAddress string, activationLink string) (response bool, err error) {
+	fromAddress := os.Getenv("EMAİL")
```

Comment:
burada her mail gönderilmek istendiğinde env den okumaya çalışıcaz bu performans açısında çok kötü bunları bir kere okuyup kullanmalıyız. buraya mail için bir builder olmalı. ayrıca bu bir mail func'u kafka ile beraber durmamalı. bir de typo olmuş "EMAİL"